dot/core/test_helpers.go

```diff
@@ -105,6 +107,13 @@ func NewTestService(t *testing.T, cfg *Config) *Service {
 		rtCfg.CodeHash, err = cfg.StorageState.LoadCodeHash(nil)
 		require.NoError(t, err)
 
+		nodeStorage := runtime.NodeStorage{}
+
+		nodeStorage.BaseDB, err = utils.SetupDatabase(filepath.Join(testDatadirPath, "offline_storage"), false)
```

Comment:
would be preferred to use `stateSrvc.BaseDB` here if `stateSrvc != nil`

Reply:
Updated.